zng/docs/spec.md

```diff
@@ -467,59 +500,57 @@ i.e., via `\u{0a}` or `\x0a`.
 A line that begins with `#` is a control message and all other lines
 are values.
 
-### 3.1 ZNG Control Messages
+### 4.1 Control Messages
 
-ZNG control messages have one of four forms defined below.
+TZNG control messages have one of four forms defined below.
 
 Any line beginning with `#` that does not conform with the syntax described here
 is an error.
-When errors are encountered parsing ZNG, an implementation should return a
-corresponding error and allow ZNG parsing to proceed if desired.
+When errors are encountered parsing TZNG, an implementation should return a
+corresponding error and allow TZNG parsing to proceed if desired.
 
-### 3.1.1 Type Binding
+### 4.1.1 Type Binding
 
-A type binding has the following form:
+A TZNG type binding has the following form:
 ```
 #<type-tag>:<type-string>
 ```
 Here, `<type-tag>` is a string decimal integer and `<type-string>`
-is a string defining a type according to the ZNG type syntax creates a binding
-between the indicated tag and the indicated type.
+is a string defining a type according to the ZNG type syntax. They create
+a binding between the indicated tag and the indicated type.
 
-### 3.1.2 Type Alias
+### 4.1.2 Type Alias
 
-A type alias has the following form:
+A TZNG type alias has the following form:
 ```
 #<type-name>:<type-string>
 ```
 Here, `<type-name>` is an identifier and `<type-string>`
-is a string defining a type according to the ZNG type syntax creates a binding
-between the indicated tag and the indicated type.
+is a string defining a type according to the ZNG type syntax. They create a
+binding between the indicated tag and the indicated type.
 This form defines an alias mapping the identifier to the indicated type.
-`<type-name>` is an identifier with semantics as defined in Section 2.1.1.4.
+`<type-name>` is an identifier with semantics as defined in [Section 3.1.1.5](#3115-alias-typedef).
 
-It is an error to define an alias that has the same name as a primitive type.
-It is also an error to redefine a previously defined alias with a
-type that differs from the original definition.
 
-### 3.1.3 Application-specific Payload
+### 4.1.3 Application-Specific Payload
 
-An application-specific payload has the following form:
+A TZNG application-specific payload has the following form:
 ```
-#!<control code>:<payload>
+#!<control-code>:<payload>
 ```
-Here, `<control code>` is a decimal integer in the range 6-127 and `<payload>`
+Here, `<control-code>` is a decimal integer in the range 6-127 and `<payload>`
```

Comment:
I noticed that the majority of our other `<things-in-angle-brackets>` use hyphens in place of spaces, so I've made that consistent here.